examples/demo/gloo/molecular_viewer.py

```diff
@@ -74,7 +74,7 @@
     pos.z += v_radius*z;
     vec3 pos2 = pos.xyz;
     pos = u_projection * pos;
-    gl_FragDepth = 0.5*(pos.z / pos.w)+0.5;
+//    gl_FragDepth = 0.5*(pos.z / pos.w)+0.5;
```

Comment:
Is there a reason for changing only one of these? (And why comment, not just remove?)